test/graphql/AdminTask/queries/getAdminTasks.test.ts

```diff
@@ -103,66 +130,102 @@ describe("getAdminTasks", () => {
   it("returns an empty array if no adminTaskTypes are provided", async () => {
     const result = await getAdminTasks({
       adminTaskTypes: [],
-      statuses: [ApprovalStatus.pending]
+      statuses: [ApprovalStatus.pending],
+      secretary: admin.secretary
     });
     expect(result).toEqual({ results: [], shouldFetchMore: false });
   });
 
   it("returns only pending companies", async () => {
     await expectToFindAdminTaskWithStatuses(
       [pendingCompany],
-      [ApprovalStatus.pending]
+      [ApprovalStatus.pending],
+      admin.secretary
     );
   });
 
   it("returns only approved companies", async () => {
     await expectToFindAdminTaskWithStatuses(
       [approvedCompany],
-      [ApprovalStatus.approved]
+      [ApprovalStatus.approved],
+      admin.secretary
     );
   });
 
   it("returns only rejected companies", async () => {
     await expectToFindAdminTaskWithStatuses(
       [rejectedCompany],
-      [ApprovalStatus.rejected]
+      [ApprovalStatus.rejected],
+      admin.secretary
     );
   });
 
   it("returns only pending applicants", async () => {
     await expectToFindAdminTaskWithStatuses(
       [pendingApplicant],
-      [ApprovalStatus.pending]
+      [ApprovalStatus.pending],
+      admin.secretary
     );
   });
 
   it("returns only approved applicants", async () => {
     await expectToFindAdminTaskWithStatuses(
       [approvedApplicant],
-      [ApprovalStatus.approved]
+      [ApprovalStatus.approved],
+      admin.secretary
     );
   });
 
   it("returns only rejected applicants", async () => {
     await expectToFindAdminTaskWithStatuses(
       [rejectedApplicant],
-      [ApprovalStatus.rejected]
+      [ApprovalStatus.rejected],
+      admin.secretary
+    );
+  });
+
+  it("returns only pending offers", async () => {
+    await expectToFindAdminTaskWithStatuses(
+      [pendingOffer],
+      [ApprovalStatus.pending],
+      admin.secretary
+    );
+  });
+
+  it("returns only approved offers", async () => {
+    await expectToFindAdminTaskWithStatuses(
+      [approvedOffer],
+      [ApprovalStatus.approved],
+      admin.secretary
+    );
+  });
+
+  it("returns only rejected offers", async () => {
+    await expectToFindAdminTaskWithStatuses(
+      [rejectedOffer],
+      [ApprovalStatus.rejected],
+      admin.secretary
     );
   });
 
   it("returns only pending applicants and companies", async () => {
     await expectToFindAdminTaskWithStatuses(
       [pendingApplicant, pendingCompany],
-      [ApprovalStatus.pending]
+      [ApprovalStatus.pending],
+      admin.secretary
     );
   });
 
-  it("sorts all applicants and companies in any status by updated timestamp", async () => {
+  it("sorts all applicants, companies and offer in any status by updated timestamp", async () => {
```

Comment:
aca faltaría poner  `offer` en plural:
`sorts all applicants, companies and offers in any status by updated timestamp`